### PR TITLE
feat: nft collection sort by timestamp

### PR DESCRIPTION
**Pull request by zmcNotafraid**
issue: https://github.com/Magickbase/ckb-explorer-public-issues/issues/664

### CODE REVIEW

File: src/pages/NftCollections/List.tsx [hunk 1]
```diff
@@ -113,6 +114,23 @@ const HolderMinterSort = () => {
...
+  sortField: 'transactions' | 'timestamp'
+  fieldI18n: string
+}
+const SimpleSortHeader: React.FC<SimpleSortProps> = ({ sortField, fieldI18n }) => {
+  const sortParam = useNFTCollectionsSortParam()
+  const { handleSortClick } = sortParam
+  return (
+    <span>
+      {/* eslint-disable-next-line jsx-a11y/click-events-have-key-events */}
+      <span onClick={() => handleSortClick(sortField)} role="button" tabIndex={0}>
```
Comment (Keith-CY) [inline src/pages/NftCollections/List.tsx:127]: I would suggest using a link leading to `/xxx?sorted_by=xxx`, e.g. `/nft-collection?sorted_by=created_time.asc`. By doing so, the ordered list can be shared by URL.

Comment (Keith-CY) [reply src/pages/NftCollections/List.tsx:127]: But don't miss other url queries in the url

Comment (Keith-CY) [reply src/pages/NftCollections/List.tsx:127]: I found an existing hook for `sort` at https://github.com/nervosnetwork/ckb-explorer-frontend/blob/develop/src/hooks/route.ts#L22

Comment (zmcNotafraid) [reply src/pages/NftCollections/List.tsx:127]: I think this feature is support now. When I click the sort, the url change to  `http://localhost:3000/en/nft-collections?sort=timestamp.desc`

File: src/pages/NftCollections/List.tsx [hunk 2]
```diff
@@ -113,6 +114,23 @@ const HolderMinterSort = () => {
     </div>
   )
 }
+interface SimpleSortProps {
+  sortField: 'transactions' | 'timestamp'
+  fieldI18n: string
+}
+const SimpleSortHeader: React.FC<SimpleSortProps> = ({ sortField, fieldI18n }) => {
+  const sortParam = useNFTCollectionsSortParam()
```
[the remark this answers is not in the document]
Comment (zmcNotafraid) [reply src/pages/NftCollections/List.tsx:122]: Please give me a code suggestion

**Review by Keith-CY — APPROVED** [review]

**Comment by Keith-CY** [comment]
image
The placeholder of loading could be updated for one more column

**Comment by FrederLu** [comment]
image

`Creation time` sorting logic needs to consider `type` sorting.
image

Otherwise, the `Type` sorting and actual data will be abnormal.

**Comment by zmcNotafraid** [comment]
> image The placeholder of loading could be updated for one more column

Resolved

**Review by FrederLu — APPROVED** [review]

**Pull request merged**